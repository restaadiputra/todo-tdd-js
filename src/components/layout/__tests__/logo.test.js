import React from 'react';
import Logo from '../logo';
import { renderWithRouter } from 'test/utils';

test('render without error', () => {
  const { container, getByText } = renderWithRouter(<Logo />);
  expect(getByText(/simple todo/i)).toBeInTheDocument();
  expect(container.firstChild.getAttribute('href')).toBe('/');
  expect(container.firstChild).toMatchSnapshot();
});
