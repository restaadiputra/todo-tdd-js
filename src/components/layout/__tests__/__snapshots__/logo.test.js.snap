// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render without error 1`] = `
<a
  aria-current="page"
  class="p-2 mr-4 inline-flex items-center active"
  href="/"
>
  <svg
    class="clipboard w-6 h-6 text-white mr-2"
    fill="currentColor"
    viewBox="0 0 20 20"
  >
    <path
      d="M8 3a1 1 0 011-1h2a1 1 0 110 2H9a1 1 0 01-1-1z"
    />
    <path
      d="M6 3a2 2 0 00-2 2v11a2 2 0 002 2h8a2 2 0 002-2V5a2 2 0 00-2-2 3 3 0 01-3 3H9a3 3 0 01-3-3z"
    />
  </svg>
  <span
    class="text-xl text-white font-bold uppercase tracking-wide"
  >
    Simple Todo
  </span>
</a>
`;
